apis/v1/gateway_types.go

```diff
@@ -483,6 +483,14 @@ type AllowedRoutes struct {
 	//
 	// +optional
 	// +kubebuilder:validation:MaxItems=8
+	//
+	// The following list states the Route kinds compatible for each protocolType
```

Comment:
One last change, folks are allowed to make their own Route types, and their own protocolTypes, so let's update this wording to be about included ones.

```suggestion
	// The following list states the Route kinds compatible for each protocolType for
	// Route kinds and protocolTypes included in Gateway API
```